tests/io/test.xpi.js

```diff
@@ -425,34 +429,33 @@ describe('Xpi.getFileAsStream()', function getFileAsStreamCallback() {
 
     this.openReadStreamStub.yields(null, rstream);
 
-    return myXpi.getFileAsStream('manifest.json')
-      .then((readStream) => {
-        return new Promise((resolve, reject) => {
-          let chunks = '';
-          readStream
-            .on('readable', () => {
-              let chunk;
-              // eslint-disable-next-line no-cond-assign
-              while ((chunk = readStream.read()) !== null) {
-                chunks += chunk.toString();
-              }
-            })
-            .on('end', () => {
-              resolve(chunks);
-            })
-            .on('error', (err) => {
-              reject(err);
-            });
+    const readStream = await myXpi.getFileAsStream('manifest.json');
+
+    const onceReadString = new Promise((resolve, reject) => {
```

Comment:
we should be able to reuse the `readStringFromStream` test helper from #1733

Reply:
@rpl I can't make this change until #1733 is merged (because there I included readStringFromStream into helpers)

Reply:
@rpl I've made necessary changes. Please check it out.